src/shared/base-filesystem.c

```diff
@@ -45,13 +45,14 @@ static const BaseFilesystem table[] = {
          * been mounted into) it is thus necessary to create a symlink pointing to the right subdirectory of
          * /usr/ first — otherwise we couldn't invoke any dynamic binary. Let's detect this case here, and
          * create the symlink as needed should it be missing. We prefer doing this consistently with Debian's
-         * multiarch logic, but support Fedora-style multilib too. */
+         * multiarch logic, but support Fedora-style and Arch-style multilib too. */
```

Comment:
well Arch doesn't do multilib, right? As I just was informed, on Arch all of /lib/, /lib64/ and /usr/lib64 are symlinks to /usr/lib/? That would mean Arch doesn't do multilib at all, nor multiarch, but only native arch.

This should be reworded in the comment then.

maybe something along the lines of "Debian's multiarch logic, Fedora-style multilib, and Arch's simple scheme that does neither"?

Reply:
Arch has something under the name [multilib](https://wiki.archlinux.org/title/Official_repositories#multilib), everything from there ending up in a separate `/usr/lib32`, that is not a symlink to anything but a real directory (there is also no `/lib32` at the filesystem root).

Reply:
ok, so if they call their "inverted" multilib actually "multilib" then the comment actually sounds OK. seems good to merge then.